<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Surefire suite]" verbose="0">
  <test thread-count="5" name="Surefire test(failed)" verbose="0">
    <classes>
      <class name="RahuleShettySeleniumCourse.seleniumframeworkdesign.LoginErrorValidationTest">
        <methods>
          <include name="LoginErrorValidation"/>
          <include name="closeBrowser"/>
          <include name="launchApp"/>
          <include name="productAddedToCartTest"/>
        </methods>
      </class> <!-- RahuleShettySeleniumCourse.seleniumframeworkdesign.LoginErrorValidationTest -->
      <class name="RahuleShettySeleniumCourse.seleniumframeworkdesign.SubmitOrderTest">
        <methods>
          <include name="closeBrowser"/>
          <include name="submitOrder" invocation-numbers="0"/>
          <include name="launchApp"/>
          <include name="orderHistoryTest"/>
        </methods>
      </class> <!-- RahuleShettySeleniumCourse.seleniumframeworkdesign.SubmitOrderTest -->
    </classes>
  </test> <!-- Surefire test(failed) -->
</suite> <!-- Failed suite [Surefire suite] -->
